libsolidity/formal/CHC.cpp

```diff
@@ -838,7 +838,7 @@ void CHC::internalFunctionCall(FunctionCall const& _funCall)
 	connectBlocks(
 		m_currentBlock,
 		predicate(*m_errorDest),
-		errorFlag().currentValue() > 0
+		errorFlag().currentValue() > 0 && currentPathConditions()
```

Comment:
So the idea is to extend the query with the current path conditions every time we shortcut a function because of a target? Yea I guess that makes sense

Reply:
Yes. I did not find a way to "call" the function conditionally, but this way we do it one step later, i.e., the error from a function call is propagated only if the path condition for the function call to occur holds.